webhooks/common.go

```diff
@@ -85,10 +85,23 @@ func NewFeatureFlagSourceSpec(env types.EnvConfig) *api.FeatureFlagSourceSpec {
 	}
 }
 
+func shouldUseRPC(annotations map[string]string) bool {
```

Comment:
nit since also in-process is over rpc
```suggestion
func shouldUseSidecar(annotations map[string]string) bool {
```

Reply:
I noticed that we have `handleRPCEvaluation` and I suggest changing to something more fitting like `handleSidecarInjection`

Reply:
I agree with this. I find that `handleRPCEvaluation` confusing, because at the time this runs, we're not evaluating anything.